packages/ui/v2/core/VersionInfo.tsx

```diff
@@ -0,0 +1,13 @@
+import { COMPANY_NAME, CalComVersion } from "@calcom/lib/constants";
```

Comment:
Can you please add this component outside of the v2 folder? New components can be added inside `packages/ui/components`. We try to get rid of v2 asap.

Reply:
i dont quite understand how I add it in ui/components

do i need an index.tsx?